packages/clarity-js/src/core/measure.ts

```diff
@@ -13,6 +14,7 @@ export default function (method: Function): Function {
         if (duration > config.longTask) {
             metric.count(Metric.LongTaskCount);
             metric.max(Metric.ThreadBlockedTime, duration);
+            internal.log(Code.FunctionExecutionTime, Severity.Info, `${method.displayName || method.name}-${duration}`);
```

Comment:
can we give these functions ids rather than full string names (like 1,2,3,etc). we are trying to reduce the size of clarity.js

Reply:
also shorten displayName please

Reply:
done